pigar/cmd.py

```diff
@@ -84,13 +84,21 @@ def parse_args(args=None):
         default=['=='],
         help='The comparison operator for versions, alternatives: [==, ~=, >=]'
     )
+    parser.add_argument(
+        '--without-referenced-commits',
```

Comment:
typo 😅

Reply:
@damnever I am sorry I do not see the typo. Did you fix it already?

Reply:
`commits` -> `comments`?

Reply:
@damnever Thank you, I see now. I will fix it.